packages/loader/container-loader/src/loader.ts

```diff
@@ -173,9 +182,17 @@ export interface ILoaderServices {
     readonly documentServiceFactory: IDocumentServiceFactory;
     /**
      * The code loader handles loading the necessary code
-     * for running a container once it is loaded.
+     * for running a container once it is loaded.  Exactly one of
+     * codeLoader or runtimeCallback must be provided.
+     */
+    readonly codeLoader?: ICodeLoader;
+
+    /**
+     * The runtimeCallback can be used to provide a runtime factory for
+     * static loading instead of the codeLoader.  Exactly one of
+     * codeLoader or runtimeCallback must be provided.
      */
-    readonly codeLoader: ICodeLoader;
+    readonly runtimeCallback?: (q: IQuorum) => IRuntimeFactory;
```

Comment:
i don't really see the advantage here why is:
runtimeCallback: (q: IQuorum) => {return IRuntimeFactory;}
better than 
codeLoader: {load(codeDetails)=>{return IRuntimeFactory;}}

Reply:
you can accomplish the same thing by just adding a static create to the Loader, or even extending the load via a new class, and call the callback in load. this seems better that diverging the internals